.github/workflows/coverage.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     steps:
       - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683
-      - uses: systemd/mkosi@660874417bbe355121d705d2a049af39ddf28b58
+      - uses: systemd/mkosi@600d847f59ca78dd04a3a4127debfb934e46fc2e
```

Comment:
BTW, fetch-mkosi script should be update to make it also update the commit hash in coverage.yml.